dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -198,3 +205,9 @@ http.proxyredirect.httpsPort.desc=The (secure) port to redirect to. Incoming HTT
 
 http.autoDecompression=Auto decompression
 http.autoDecompression.desc=Specifies whether the HTTP Channel automatically decompresses incoming request body data.
+
+http.compression.types=List of content types for response compression
+http.compression.types.desc=Specifies a list of content types for which the HTTP Channel will perform compression of the response if able.
+
+http.compression.serverPreferredAlgorithm=Server preferred algorithm for compression
+http.compression.serverPreferredAlgorithm.desc=Specifies the server's preferred compression algorithm. The configured compression algorithm will be used to compress the body of responses when specified with a non-zero quality value in the request's Accept-Encoding header. The valid compression algorithms include: deflate, gzip, x-gzip, zlib, and identity.
```

Comment:
**change**
The configured compression algorithm will be used to compress the body of responses when specified with a non-zero quality value in the request's Accept-Encoding header. 
**to**
The configured compression algorithm is used to compress the body of responses when it is specified with a non-zero quality value in the request's Accept-Encoding header.